expo/expo
# [fingerprint] spawn ExpoConfigLoader with the projectRoot cwd

# Why

Using `__dirname` as `cwd` breaks config loading in configs that e.g. rely on `ts-node/register`.
See https://github.com/expo/expo/issues/29793 for more context.

# How

Don't override the `cwd` when spawning the `ExpoConfigLoader.js` node process. It should persist the `cwd` from the parent process.

# Test Plan

1. Clone the reproduction repo: https://github.com/a-gierczak/expo-fingerprint-tsnode-bug-repro
2. Run `yarn install @expo/fingerprint` to install it locally
3. Run `yarn fingerprint > /path/to/repo`
4. It should produce the `Cannot get Expo config from an Expo project` error
5. `yarn link` the local (fixed) version
6. `yarn add @expo/fingerprint` again, to use the locally-linked version
7. Run  `yarn fingerprint > /path/to/repo` again
8. It should show the fingerprint without errors

# Checklist
